.github/workflows/security.yml

```diff
@@ -0,0 +1,33 @@
+name: Security
+
+on:
+  pull_request: {}
+  push:
+    branches: [ "main" ]
+  schedule:
+    - cron: "30 0 1,15 * *"
```

Comment:
Should this be running on a cron?

Reply:
Yes, as otherwise it would only trigger when there is activity on this repo. But in case there isn't any, we still want to trigger this security check so we can get alerted if any dependencies have had vulnerabilities discovered.

Reply:
Thanks for the clarification.

Reply:
Shouldn't that be done by Dependabot and Snyk anyway?

Reply:
That's done **also** by Dependabot and Snyk, correct. From my understanding we need both snyk, semgrep and govulncheck to maximize the chance of being alerted on issues, so we don't have just a single point of failure for security checks. 

I'll cc: @evansims and @poovamraj tho to confirm.

Reply:
👋 Yup! Snyk is excellent, but its handling of Go has not proven to be the most reliable in our past experiences. On the other hand, Govulncheck has proven quite reliable. Having it as an additional security filter here makes sense. 👍

Reply:
Thanks @evansims!